server/service/src/programs/encounter/insert.rs

```diff
@@ -138,7 +150,7 @@ fn generate(
         r#type: input.r#type.clone(),
         data: input.data,
         form_schema_id: Some(input.schema_id),
-        status: DocumentStatus::Active,
+        status,
```

Comment:
I also set the document status to deleted. This comes in handy for the dashboard reports because then we only need to filter out the deleted document once instead of for every report...